resume-project/src/App.js

```diff
@@ -2,8 +2,103 @@ import React from 'react';
 
 const App = () => (
   <div>
-    test!
-  </div>
+    <section className="container">
+      <section id="left-side">
```

Comment:
can you remove the id's? try to move them to classes React doesn't play nice w/ id's